assets/js/base/components/cart-checkout/shipping-rates-control/packages.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Internal dependencies
+ */
+import ShippingRatesControlPackage from '../shipping-rates-control-package';
+
+/**
+ * @typedef {import('react')} React
+ */
+
+/**
+ * Renders multiple packages within the slotfill.
+ *
+ * @param {Object} props Incoming props.
+ * @param {Array} props.packages Array of packages.
+ * @param {React.ReactElement} props.noResultsMessage Rendered when there are no rates in a package.
+ * @param {boolean} props.collapsible If the package should be rendered as a
+ * collapsible panel.
+ * @param {boolean} props.collapse If the panel should be collapsed by default,
+ * only works if collapsible is true.
+ * @param {boolean} props.showItems If we should items below the package name.
+ * @param {Function} [props.renderOption] Function to render a shipping rate.
+ * @return {React.ReactElement|null} Rendered components.
+ */
+const Packages = ( {
```

Comment:
I'm not sure there is need to move this component out into it's own file since it's not really being imported into multiple places (and at a minimum could have been considered separately maybe?). Having it directly in context of where it is consumed could be useful. If/when we move things like this into their own files it'd probably be good to create the file in `.ts` too :). With that said, this is a non-blocking comment.

Reply:
There was a reason I did this, I can't quite remember right now but I think it was because this component is exported as well as the component in the index file, and having both default exports from one file was difficult to follow. I can revert and see what breaks if needed. I don't want to convert to TS in this PR.

Reply:
Done in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4025/commits/e362f1c542aaced890f954780aae9d270e05fada